packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -101,7 +101,7 @@ export interface IProject extends IResource, iam.IGrantable, ec2.IConnectable {
    * @param metricName The name of the metric
    * @param props Customization properties
    */
-  metric(metricName: string, props: cloudwatch.MetricOptions): cloudwatch.Metric;
+  metric(metricName: string, props?: cloudwatch.MetricOptions): cloudwatch.Metric;
```

Comment:
Always prefer this (to avoid null derefs):

```suggestion
  metric(metricName: string, props: cloudwatch.MetricOptions = { }): cloudwatch.Metric;
```

Reply:
This is the interface signature, default argument is an implementation declaration.